resources/default_pipeline_environment.yml

```
@@ -493,7 +493,7 @@ steps:
       dockerName: 'bundler'
       dockerWorkspace: ''
   slackSendNotification:
-    color: "${buildStatus == 'SUCCESS'?'#008000':'#E60000'}"
+    color: "${['SUCCESS': '#8cc04f', 'FAILURE': '#d54c53', 'ABORTED': '#949393', 'UNSTABLE': '#f6b44b', 'PAUSED': '#24b0d5', 'UNKNOWN': '#d54cc4'].get(currentBuild.result, '#d54cc4')}"
```

The template passes in `currentBuild.result` as `buildStatus`

```suggestion
    color: "${['SUCCESS': '#8cc04f', 'FAILURE': '#d54c53', 'ABORTED': '#949393', 'UNSTABLE': '#f6b44b', 'PAUSED': '#24b0d5', 'UNKNOWN': '#d54cc4'].get(buildStatus, '#d54cc4')}"
```